### PR TITLE
Tables for algorithm parameters - QGIS Layer tools

### PR DESCRIPTION
**Pull request by havatv**
Goal:

Ticket(s): #4429

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: source/docs/user_manual/processing_algs/qgis/layertools.rst [hunk 1]
```diff
@@ -27,11 +27,33 @@ The output layer contains a single bounding box for the whole input layer.
...
+   :widths: 20 20 20 40
+   :stub-columns: 0
+
+   *  - Label
+      - Name
+      - Type
+      - Description
+   *  - **Layer**
+      - ``INPUT``
+      - [layer:any]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/layertools.rst]:
```suggestion
      - [layer]
```

File: source/docs/user_manual/processing_algs/qgis/layertools.rst [hunk 2]
```diff
@@ -27,11 +27,33 @@ The output layer contains a single bounding box for the whole input layer.
...
+      - Input vector layer
+
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/layertools.rst]: is `output` not a parameter ? and there's a round parameter also in the alg

Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/layertools.rst]: Yes! Good to have you around! I have fixed it in the latest commit.

File: source/docs/user_manual/processing_algs/qgis/layertools.rst (hunk 1 above)
Comment (havatv) [reply source/docs/user_manual/processing_algs/qgis/layertools.rst]: Yes! I fixed this with the rest in the latest commit,

File: source/docs/user_manual/processing_algs/qgis/layertools.rst (hunk 2 above)
Comment (DelazJ) [reply source/docs/user_manual/processing_algs/qgis/layertools.rst]: > Yes! Good to have you around!

To have you, too :)

**Comment by havatv** [comment]
Thanks again, @DelazJ ! I was too quick on this one...

**Pull request merged**